faq.rst

```diff
@@ -243,6 +243,7 @@ Simply use ``git checkout`` to checkout another branch in the current directory:
    $ git branch
      master
    * 3.5
+   $ make
```

Comment:
I mentioned that in the commit message.  You can't do any testing unless you run make after switching branches.  If you forget the make, weird and mysterious things may happen.